### PR TITLE
added sign in with Google feature

### PR DESCRIPTION
**Pull request by qevyn**
Due to the deprecation of the Javascript based Google Platform Library, I migrated the code to use the new Google Identity Services instead.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Selecto-env/
+myenv/
```
Comment (jeffery-do) [inline .gitignore:1]: we can just have a bunch of these.

Ideally folks should e putting this folder within their internal .git/.gitignore

File: web/selecto/products/forms.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+from django import forms
+
+class CustomUserCreationForm(UserCreationForm):
+    email = forms.EmailField(required=True)
+
+    class Meta:
+        model = User
+        fields = ("username", "email", "password1", "password2")
```
Comment (jeffery-do) [inline web/selecto/products/forms.py:10]: 2 passwords? Why?

File: web/selecto/selecto/settings.py [hunk 3]
```diff
@@ -52,6 +61,21 @@
...
+
+
+SOCIALACCOUNT_PROVIDERS = {
+    'google': {
+        'SCOPE': ['profile', 'email'],
+        'AUTH_PARAMS': {'access_type': 'online'},
+        'OAUTH_PKCE_ENABLED': True,
+        'APP': {
+            'client_id': os.getenv('CLIENT_ID'),
+            'secret': os.getenv('SECRET_KEY'),
```
Comment (jeffery-do) [inline web/selecto/selecto/settings.py:73]: These configs should be GOOGLE specific. So should be named something like GOOGLE_CLIENT_ID, GOOGLE_SECRET_KEY

**Review by jeffery-do — DISMISSED** [review]
Overall, lgtm.

Some things need to be changed, but I can address some of them in another PR.

**Review by shane-co — APPROVED** [review]
lgtm

**Pull request merged**